src/Compilers/CSharp/Portable/Symbols/TypeParameterSymbol.cs

```diff
@@ -444,15 +444,14 @@ internal bool IsReferenceTypeFromConstraintTypes(ImmutableArray<TypeSymbolWithAn
             return AnyConstraintTypes(constraintTypes, inProgress, (type, arg) => ConstraintImpliesReferenceType(type.TypeSymbol, arg));
         }
 
-        internal bool? IsNotNullableIfReferenceTypeFromConstraintTypes(ImmutableArray<TypeSymbolWithAnnotations> constraintTypes, ConsList<TypeParameterSymbol> inProgress)
+        internal bool? IsNotNullableIfReferenceTypeFromConstraintTypes(ImmutableArray<TypeSymbolWithAnnotations> constraintTypes)
```

Comment:
>internal [](start = 8, length = 8)

`static`

Reply:
> static

Will follow up on this in the next  PR

---
In reply to: [217498379](https://github.com/dotnet/roslyn/pull/29809#discussion_r217498379) [](ancestors = 217498379)